django-backend/fecfiler/committee_accounts/utils.py

```diff
@@ -105,14 +105,29 @@ def get_committee_account_data(committee_id):
 """
 
 
-def query_fec_api(endpoint, params):
+def get_environment():
```

Comment:
I'm wondering if we can add a dedicated env var for this to have better control.  I feel like it could be easy to miss updating this if we changed/got rid of the timeout cookie.